### PR TITLE
HWPV-176: Migrate unit tests to jest

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

File: test/integration/services/data/test-get-individual-claim-details.js
Comment (tpmcgowan) [inline test/integration/services/data/test-get-individual-claim-details.js:1]: Should this all be commented out?

File: test/integration/services/data/test-submit-claim-response.js
Comment (tpmcgowan) [inline test/integration/services/data/test-submit-claim-response.js:1]: This one all commented out too?

File: test/integration/services/data/test-get-individual-claim-details.js
Comment (psoleckimoj) [reply test/integration/services/data/test-get-individual-claim-details.js:1]: Actually they're integration tests. I've spotted some more so I'll see.

Comment (tpmcgowan) [reply test/integration/services/data/test-get-individual-claim-details.js:1]: It passes though 😂

File: test/integration/services/data/test-submit-claim-response.js
Comment (psoleckimoj) [reply test/integration/services/data/test-submit-claim-response.js:1]: Still an int test

**Comment by psoleckimoj** [comment]
@tpmcgowan will sort the int tests when we get onto that.

**Review by tpmcgowan — APPROVED** [review]
LGTM!

**Pull request merged**